pkg/nvidiagpu/precompiled.go

```
@@ -0,0 +1,259 @@
+	glog.V(100).Infof("No short-form driver version found, using: %s", driverVersions[0])
+	return driverVersions[0], nil
+}
+
+func listRegistryTags(authBase64 string) ([]string, error) {
+	tagsURL := fmt.Sprintf("https://%s/v2/%s/tags/list", precompiledRegistry, precompiledRepository)
+
+	resp, err := http.Get(tagsURL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to contact registry: %w", err)
+	}
+	resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)